src/applications/mhv/landing-page/selectors/isLandingPageEnabledForUser.js

```diff
@@ -19,14 +19,16 @@ import FEATURE_FLAG_NAMES from '~/platform/utilities/feature-toggles/featureFlag
  * current user. Returns false, otherwise.
  */
 export const isLandingPageEnabledForUser = state => {
+  let enabledForUser = false;
   const loggedIn = isLoggedIn(state);
-  if (!loggedIn) return false;
   const mhvlpFeatureToggle = FEATURE_FLAG_NAMES.mhvLandingPageEnabled;
   const featureToggleEnabled = toggleValues(state)[mhvlpFeatureToggle];
-  if (!featureToggleEnabled) return false;
   // selectPatientFacilites alters the facilities array using map. You _must_
   // use this selector. DO NOT traverse state. e.g. - state.user.profile.facilities
   const facilities = selectPatientFacilities(state) || [];
   const hasFacilities = facilities.length > 0;
-  return loggedIn && featureToggleEnabled && hasFacilities;
+  if (featureToggleEnabled && loggedIn && hasFacilities) {
```

Comment:
Love this reorg with just one return! Thx

Reply:
thanks! will give it a try. i think i did something like that initially, but the linter complained.

Reply:
perhaps something like `return !!(featureToggleEnabled && loggedIn && hasFacilities);` would work but for readability we might want to stick with your addition.

Reply:
I believe the linter was complaining about the combination of guard clauses (if..return) and a final return statement that included the same conditions. Now that you've removed the guard clauses on lines 23 & 26, you should be able to keep  what was on line 31 without the linter giving you any guff.